RUMM-1799 Clarify Linking Crash Reporting Using SPM (#707)
DataDog/dd-sdk-ios: docs/rum_collection/crash_reporting.md

### Add crash reporting 

> If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [iOS RUM setup documentation][2].

Reviewer: ```suggestion
If you have not set up the SDK yet, follow the [in-app setup instructions][1] or see the [iOS RUM setup documentation][2].
```

#### Dependency Manager
{{< tabs >}}
{{% tab "CocoaPods" %}}
Add `DatadogSDKCrashReporting` to your `Podfile`:
```ruby
platform :ios, '11.0'
use_frameworks!

target 'App' do
  pod 'DatadogSDKCrashReporting'
end
```
{{% /tab %}}
{{% tab "Swift Package Manager" %}}

> Add the Package at `https://github.com/DataDog/dd-sdk-ios` then link the `DatadogCrashReporting` library to your application target. 

Reviewer: ```suggestion
Add the package at `https://github.com/DataDog/dd-sdk-ios` and link `DatadogCrashReporting` to your application target. 
```

> > If your already linking `Datadog` or `DatadogStatic` library, replace it with `DatadogCrashReporting`.

Author: How can I mark this line as 'important', highlighting somehow?

Reviewer: ```suggestion

**Note:** If you link to `Datadog` or the `DatadogStatic` library, replace it with `DatadogCrashReporting`.
```
